perf/benchmark/runner/runner.py

```diff
@@ -189,9 +190,9 @@ def ingress(self, load_gen_cmd):
     def execute_sidecar_mode(self, sidecar_mode, load_gen_type, load_gen_cmd, sidecar_mode_func, labels, perf_label_suffix):
         print('-------------- Running in {sidecar_mode} mode --------------'.format(sidecar_mode=sidecar_mode))
         if load_gen_type == "fortio":
-            kubectl_exec(self.client.name, sidecar_mode_func(load_gen_cmd, perf_label_suffix))
+            kubectl_exec(self.client.name, sidecar_mode_func(load_gen_cmd, sidecar_mode))
         elif load_gen_type == "nighthawk":
-            run_nighthawk(self.client.name, sidecar_mode_func(load_gen_cmd, perf_label_suffix), labels + perf_label_suffix)
+            run_nighthawk(self.client.name, sidecar_mode_func(load_gen_cmd, sidecar_mode), labels + "_" + sidecar_mode)
 
         if self.perf_record and len(perf_label_suffix) > 0:
```

Comment:
@oschaaf we are appending perf_label_suffix, which starts with `srv` here. This would be helpful info for your flamegraph PR.